core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -221,6 +224,10 @@ public final class ServerBuilder implements TlsSetters, ServiceConfigsBuilder<Se
     private int http1MaxChunkSize = Flags.defaultHttp1MaxChunkSize();
     private int proxyProtocolMaxTlvSize = PROXY_PROTOCOL_DEFAULT_MAX_TLV_SIZE;
     private GracefulShutdown gracefulShutdown = GracefulShutdown.disabled();
+    private GracefulShutdown workerGroupGracefulShutdown = GracefulShutdown.disabled();
+    private GracefulShutdown bossGroupGracefulShutdown = GracefulShutdown.disabled();
```

Comment:
Question) Would it make sense to use the default from Flags here?

e.g.
```suggestion
    private GracefulShutdown workerGroupGracefulShutdown = Flags.workerGroupGracefulShutdown();
    private GracefulShutdown bossGroupGracefulShutdown = Flags.workerGroupGracefulShutdown();
```

Reply:
I can't answer this since I'm not fully familiar with the Flags purpose. I guess it would mean that by single system property I could set defaults to both groups, which could be both handy and misleading (single configuration option for two toggles). For now, I leave current implementation in place, but will hapilly change that if you insist on the change.